### DISCUSSION TITLE
Error in clip_vit_b32 example

### DISCUSSION THREAD

**Discussion by paolobolettieri** [towhee-io/towhee#2369] [Q&A] [answered]
I am getting the following error running your "clip_vit_b32" example posted on Towhee homepage: 

"towhee/engine/operator_loader.py", line 176, in instance_operator
    return op(*arg, **kws) if kws is not None else op()
TypeError: __init__() missing 2 required positional arguments: 'device' and 'checkpoint_path'

How to fix it?

**Comment by robert-hernandez-cenk** [comment]
I'm having the same issue.

**Comment by filip-halt** [comment, marked as answer]
I believe our models were updated recently. Please delete the clip folder that is found in your operators cache (usually in user/.towhee/operators/image-text-embedding) and rerun the example. In addition to this, with the change, please change the `clip_vit_b32` to `clip_vit_base_patch32`.

**Reply by paolobolettieri** [reply]
Thank you, it works!

**Reply by jerry-mengbo** [reply]
👍🏻 it works
